### DISCUSSION TITLE
Cuphead save erro

### DISCUSSION THREAD

**Discussion by Reitonn** [skyline-emu/skyline#1829] [Q&A]
whenever i log out of cuphead my game is not saved i always have to start over

**Comment by Roguigo** [comment]
mine does the same thing, hope they fix it soon

**Reply by Aliahmad0336** [reply]
Same problem Poco x3 pro sd 860 8gb android 12

**Reply by Maty12z** [reply]
El mío hace lo mismo espero que lo arreglen pronto mi celular es un v2025 con 12de ram y un sd 732 con Android 12
